### DISCUSSION TITLE
738

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#919] [Comments]
# 738

Heap Dump를 분석하기 위해 Eclipse mat를 종종 사용한다. 너무 큰 덤프 파일을 불러오면 다음과 같이 OOM 이슈가 발생한다. An internal error occurred during: "Parsing heap dump" Out of Memory mat 에 할당된 메모리를 높여야만 대용량의 덤프 파일을 불러 올 수가 있다. 이를 위한 설정은 다음과 같다. 설정 모든 설정은 MAC OS를 기준으로 한다. mat 설정은 /Applications/mat.app/Contents/Eclipse/MemoryAnalyzer.ini 위치에 존재한다. 그래서 필요한 에디터로 (여기서는 vi) 해당 파일을 열어서 vim /Applications/mat.app/Contents/Eclipse/MemoryAn..

https://jojoldu.tistory.com/738

**Comment by jaehyeong2** [comment]
인프런에서 스프링 / 노드를 다하는건가요 아니면 따로 공부하시는건가요 향로님?

**Reply by jojoldu** [reply]
아! 팀 내부에서 검색엔진등에서 스프링을 사용하고 있습니다 ㅋㅋ
